### PR TITLE
fix(dots by default): number of dots by default appears without provi…

### PR DESCRIPTION
**Pull request by udaypydi**
…ding the number. For that the dots have to be rendered through dots boolean flag in Carousel component. For thumbnails the Dots Component can be used

- [ ] `package.json:version` has been updated with `npm version patch` (or `major/minor` as appropriate)
- [ ] `@brainhubeu/react-carousel` version has been updated in `docs-www/package.json` to the same which is in `package.json:version`

### CODE REVIEW

**Comment by piotr-s-brainhub** [comment]
Thanks @udaypydi

@Lukasz-pluszczewski what do you think about this PR?

File: src/components/Carousel.js [hunk 1]
```diff
@@ -654,14 +655,18 @@ export default class Carousel extends Component {
...
+      <div>
+        <div
+          className={classnames('BrainhubCarousel', this.getProp('className'))}
+          ref={el => this.node = el}
+        >
+          {this.renderArrowLeft()}
+          {this.renderCarouselItems()}
+          {this.renderArrowRight()}
+        </div>
+        <div className={classnames('BrainhubCarousel__dots', this.getProp('className'))}>
```
Comment (Lukasz-pluszczewski) [inline src/components/Carousel.js]: 1. The whole div should not render if `!this.getProp('dots')` as the className provided may add some margin/padding or other unexpected changes.
2. I would consider having a separate props for dots className (like `dotsClassName`). It's common to just throw some styles at the carousel without thinking that the class provided in that prop is actually applied to two things (which is quite unintuitive I think)

**Comment by Lukasz-pluszczewski** [comment]
@udaypydi Am I understand correctly that it's your intention to just wrap dots rendered automatically (when dots=true) in a container `.BrainhubCarousel__dots` and add custom class? 
I don't see any actual dots behaviour change. Or is there something missing in this PR?

**Comment by piotr-s-brainhub** [comment]
@udaypydi ?

**Comment by udaypydi** [comment]
@Lukasz-pluszczewski @piotr-s-brainhub  The pr is updated as per the review changes. Please check.

Thanks,
Uday

**Comment by udaypydi** [comment]
@Lukasz-pluszczewski @piotr-s-brainhub seems the previous pr merged has these changes. Its a redundant pr. closing it for now.
The default number of dots were fixed in my earlier PR.
Thanks

**Comment by piotr-s-brainhub** [comment]
@udaypydi 

Thanks so I close this PR.

**Pull request closed without merging**